src/tracks/playabletrack/wavetrack/ui/WaveClipTrimHandle.cpp

```diff
@@ -268,11 +372,23 @@ HitTestPreview WaveClipTrimHandle::HitPreview(const AudacityProject*, bool unsaf
 
 
 WaveClipTrimHandle::WaveClipTrimHandle(std::unique_ptr<ClipTrimPolicy>& clipTrimPolicy)
-   : mClipTrimPolicy(std::move(clipTrimPolicy))
+   : mClipTrimPolicy{ std::move(clipTrimPolicy) }
+{
+
+}
+
+void WaveClipTrimHandle::ClipTrimPolicy::Draw(TrackPanelDrawingContext&, const wxRect&, unsigned) { }
+
+wxRect WaveClipTrimHandle::ClipTrimPolicy::DrawingArea(TrackPanelDrawingContext&, const wxRect&, const wxRect&, unsigned)
 {
+   return {};
```

Comment:
See TrackPanelDrawable::DrawingArea:  default behavior should not be to return an empty wxRect, but to return the second argument unchanged.